templates/serverpod_templates/projectname_server/lib/src/greeting.spy.yaml

```diff
@@ -6,18 +6,21 @@
 # Please consult the documentation for more information on what you can add to
 # your yaml-files.
 
-
 # Name of the class to generate.
-class: Example
+class: Greeting
```

Comment:
Perhaps we should add some `###` docs here to make the documentation for the model complete.